homeassistant/components/epsonworkforce/sensor.py

```diff
@@ -10,15 +10,16 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.entity import Entity
 
-REQUIREMENTS = ['epsonprinter==0.0.8']
+REQUIREMENTS = ['epsonprinter==0.0.9']
 
 _LOGGER = logging.getLogger(__name__)
 MONITORED_CONDITIONS = {
-    'black': ['Inklevel Black', '%', 'mdi:water'],
-    'magenta': ['Inklevel Magenta', '%', 'mdi:water'],
-    'cyan': ['Inklevel Cyan', '%', 'mdi:water'],
-    'yellow': ['Inklevel Yellow', '%', 'mdi:water'],
-    'clean': ['Inklevel Cleaning', '%', 'mdi:water'],
+    'black': ['Ink level Black', '%', 'mdi:water'],
```

Comment:
This is a breaking change, since the first item will be used for entity name which will be used for entity_id.